docs/storage-abstraction.md

```diff
@@ -630,15 +573,43 @@ Put put =
         .build();
 ```
 
-##### Put with a condition
+#### `Delete` operation
+
+`Delete` is an operation to delete a record specified by a primary key.
+
+You need to create a `Delete` object first, and then you can execute the object by using the `storage.delete()` method as follows:
+
+```java
+// Create a `Delete` operation.
+Key partitionKey = Key.ofInt("c1", 10);
+Key clusteringKey = Key.of("c2", "aaa", "c3", 100L);
+
+Delete delete =
+    Delete.newBuilder()
+        .namespace("ns")
+        .table("tbl")
+        .partitionKey(partitionKey)
+        .clusteringKey(clusteringKey)
+        .build();
+
+// Execute the `Delete` operation.
+storage.delete(delete);
+```
+
+#### `Put` and `Delete` with a condition
+
+You can write arbitrary conditions (for example, a bank account balance must be equal to or more than zero) that you require a transaction to meet before being committed by implementing logic that checks the conditions in the transaction. Alternatively, you can write simple conditions in a mutation operation, such as `Put` and `Delete`.
+
+When a `Put` or `Delete` operation includes a condition, the operation is executed only if the specified condition is met. If the condition is not met when the operation is executed, an exception called `UnsatisfiedConditionException` will be thrown.
```

Comment:
I rewrote a bit this section since in the case of the storage abstraction, a `NoMutationException` is thrown instead when the condition is not satisfied.
What do you think?
 
```suggestion
You can write arbitrary conditions (for example, a bank account balance must be equal to or more than zero) that you require an operation to meet before being executed by implementing logic that checks the conditions. Alternatively, you can write simple conditions in a mutation operation, such as `Put` and `Delete`.

When a `Put` or `Delete` operation includes a condition, the operation is executed only if the specified condition is met. If the condition is not met when the operation is executed, an exception called `NoMutationException` will be thrown.
```

Reply:
LGTM! Thank you for revising this part!🙏